### PR TITLE
⚡ use HdrHistogram to aggre

### PR DESCRIPTION
**Pull request by songxiaosheng**
## Checklist
- [x] Make sure there is a [GitHub_issue](https://github.com/apache/dubbo/issues) field for the change (usually before you start working on it). Trivial changes like typos do not require a GitHub issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue.
- [ ] Each commit in the pull request should have a meaningful subject line and body.
- [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
- [ ] Check if is necessary to patch to Dubbo 3 if you are work on Dubbo 2.7
- [ ] Write necessary unit-test to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add sample in [dubbo samples](https://github.com/apache/dubbo-samples) project.
- [ ] Add some description to [dubbo-website](https://github.com/apache/dubbo-website) project if you are requesting to add a feature.
- [ ] GitHub Actions works fine on your own branch.
- [ ] If this contribution is large, please follow the [Software Donation Guide](https://github.com/apache/dubbo/wiki/Software-donation-guide).

### CODE REVIEW

**Comment by AlbumenJ** [comment]
image

**Comment by songxiaosheng** [comment]
sample as follow:
```

import org.HdrHistogram.ConcurrentDoubleHistogram;
import org.HdrHistogram.ConcurrentHistogram;

public class HistogramExample {
    public static void main(String[] args) {
        // 创建 ConcurrentHistogram 对象
        ConcurrentDoubleHistogram histogram = new ConcurrentDoubleHistogram(2);
        // 向直方图中添加数据
        histogram.recordValue(1);  // 假设添加了多个值
        histogram.recordValue(2);  // 假设添加了多个值
        histogram.recordValue(3);  // 假设添加了多个值
        histogram.recordValue(4);  // 假设添加了多个值
        histogram.recordValue(5);  // 假设添加了多个值
        histogram.recordValue(6);  // 假设添加了多个值
        histogram.recordValue(7);  // 假设添加了多个值
        histogram.recordValue(8);  // 假设添加了多个值
        histogram.recordValue(9);  // 假设添加了多个值
        histogram.recordValue(10);  // 假设添加了多个值

        double valueAtPercentile = histogram.getValueAtPercentile(80.0);
        System.out.println(valueAtPercentile);
    }
}
```
result is：
`8.0546875`

File: dubbo-metrics/dubbo-metrics-default/src/main/java/org/apache/dubbo/metrics/collector/AggregateMetricsCollector.java [hunk 1]
```diff
@@ -159,9 +159,9 @@ private void onRTEvent(RequestEvent event) {
         MethodMetric metric = new MethodMetric(applicationModel, event.getAttachmentValue(MetricsConstants.INVOCATION));
         long responseTime = event.getTimePair().calc();
         if (enableRt) {
-            TimeWindowQuantile quantile = ConcurrentHashMapUtils.computeIfAbsent(rt, metric,
-                k -> new TimeWindowQuantile(DEFAULT_COMPRESSION, bucketNum, timeWindowSeconds));
-            quantile.add(responseTime);
+            ConcurrentDoubleHistogram concurrentDoubleHistogram = ConcurrentHashMapUtils.computeIfAbsent(rt, metric,
```
Comment (AlbumenJ) [inline dubbo-metrics/dubbo-metrics-default/src/main/java/org/apache/dubbo/metrics/collector/AggregateMetricsCollector.java:162]: Is `timeWindowSeconds` here no longer needed

**Review by AlbumenJ — CHANGES_REQUESTED** [review]

File: dubbo-metrics/dubbo-metrics-default/src/main/java/org/apache/dubbo/metrics/collector/AggregateMetricsCollector.java (hunk 1 above)
Comment (songxiaosheng) [reply dubbo-metrics/dubbo-metrics-default/src/main/java/org/apache/dubbo/metrics/collector/AggregateMetricsCollector.java:162]: let me look look

Comment (songxiaosheng) [reply dubbo-metrics/dubbo-metrics-default/src/main/java/org/apache/dubbo/metrics/collector/AggregateMetricsCollector.java:162]: Sliding windows are used for aggregation statistics of minimum and maximum values. Unlike the percentile algorithm above, sliding windows use thread safe algorithms

**Comment by songxiaosheng** [comment]
After the research is completed, we will raise the PR

**Pull request closed without merging**